### ISSUE TITLE
add `gin` as `type=git autoenable=true` annex special remote?

### ISSUE THREAD

**Issue by yarikoptic** [lnnrtwttkhn/highspeed-bids#1]
ATM it requires user to "discover" (well RTFM README.md here and figure out ;-))  and add gin remote to be able to access data if cloned from this URL (and note that .gitmodules of https://git.mpib-berlin.mpg.de/wittkuhn/highspeed do point to github not gin).

RTFM could be avoided AFAIK if `gin` is added as a git annex special remote.  unfortunately we haven't added a convenience yet in datalad to do that for already existing repos, but https://github.com/datalad/datalad/issues/4697 shows how.

Then anyone who gets this dataset `datalad install`ed should get `gin` git remote automagically added  and thus make data from gin immediately available

**Comment by lnnrtwttkhn** [comment]
Hi @yarikoptic, thanks for getting in touch about this! I have thought about this as well, but didn't really know how to.

Following https://github.com/datalad/datalad/issues/4697, would it be correct to simply set:

```bash
git annex initremote gin type=git location=https://gin.g-node.org/lnnrtwttkhn/highspeed-bids.git autoenable=true 
```

Thanks!

**Comment by yarikoptic** [comment]
I think it should be sufficient (can try on a throw away clone) although I think in my example I might have missed `encryption=none` which is mandatory.  you just need to make sure you have already that repository as a git remote, which is not named `gin`.

**Comment by lnnrtwttkhn** [comment]
hey @yarikoptic, with some additional help by @mih and the team of the datalad office hour (thank you!), I managed to set this up now!

```bash
$ datalad clone https://github.com/lnnrtwttkhn/highspeed-bids.git
[INFO   ] Scanning for unlocked files (this may take some time)                                                                                                       
[INFO   ] Remote origin not usable by git-annex; setting annex-ignore                                                                                                 
[INFO   ] access to 1 dataset sibling keeper not auto-enabled, enable with:
| 		datalad siblings -d "/Users/wittkuhn/Desktop/highspeed-bids" enable -s keeper 
install(ok): /Users/wittkuhn/Desktop/highspeed-bids (dataset)
```

```bash
$ cd highspeed-bids
nrcd-osx-003854:highspeed-bids wittkuhn$ git annex info
trusted repositories: 0
semitrusted repositories: 13
	00000000-0000-0000-0000-000000000001 -- web
 	00000000-0000-0000-0000-000000000002 -- bittorrent
 	2616fba7-81bf-40cd-8812-ab38f3eca8d8 -- wittkuhn@master:/mnt/beegfs/home/wittkuhn/highspeed/highspeed-bids
 	26d3c52c-e301-451c-a2d9-79dc26db0700 -- wittkuhn@nrcd-osx-003854:~/Desktop/highspeed-bids [here]
 	39656a37-37f8-4cf9-8333-f1801b4c8024 -- dataset containing bids-converted MRI data of the highspeed project
 	40ad029e-0a11-46ed-a115-5f15b620503b -- wittkuhn@lip-osx-003854:~/Documents/highspeed/highspeed-bids
 	662bcf1d-7e57-43b2-b999-393230420e23 -- wittkuhn@lip-osx-003854:~/Documents/highspeed/highspeed-bids
 	762a1cb9-55a0-4b51-a63f-a0b85316d575 -- [gin]
 	823e52e3-ba0f-40be-afa7-25c6778a0af9 -- wittkuhn@lip-osx-003854:~/Documents/highspeed/highspeed_bids
 	abda2f25-50fd-4242-b48a-f52e94c161e5 -- wittkuhn@nrcd-osx-003854:~/Desktop/highspeed-bids
 	af0e99c7-c24c-49b1-9c4c-968a0dad9ff7 -- keeper
 	dac4a734-7a12-419a-b896-32af905e1a30 -- wittkuhn@lip-osx-003854:~/Documents/highspeed/highspeed-bids
 	f6f0765e-831b-4006-bb63-08459d8e0904 -- wittkuhn@lip-osx-003854:~/Documents/highspeed/highspeed-bids
untrusted repositories: 0
transfers in progress: none
available local disk space: 13.06 gigabytes (+1 megabyte reserved)
local annex keys: 0
local annex size: 0 bytes
annexed files in working tree: 2315
size of annexed files in working tree: 137.37 gigabytes
bloom filter size: 32 mebibytes (0% full)
backend usage: 
	MD5E: 2315
```

```bash
$ git annex get -f gin participants.tsv
get participants.tsv (from gin...) 
(checksum...) ok
(recording state in git...)
```

would close this issue, but feel free to reopen, if there are still problems!

thanks again for the help and suggestions!
